InternshipProject#2/Manager/TicketManager.cs

```diff
@@ -38,7 +38,7 @@ public async Task<TicketCreateResponse> CreateTicketAsync(TicketCreateRequest ne
             ticket.Status = 1;
             _context.Tickets.Add(ticket);
             await _context.SaveChangesAsync();
```

Comment:
You have to save only at the end.

Reply:
There is a need for 2 saves, the first save is for the ticket, so that the db generates the unique id for the ticket, then we can use the ticket.id for the watcher and for history.